docs/benchmarks/data-caching.md

```diff
@@ -73,43 +58,47 @@ The client can simultaneously use multiple servers, one server with several ip a
 
-If the scaled file is already created, but the server is not warmed up, use the following command to warm up the server:
+If the scaled file is already created, but the server is not warmed up, use the following command to warm up the server. `W` refers to the _warm up_ mode of operation.
 
-    $ ./loader -a ../twitter_dataset/twitter_dataset_30x -s docker_servers.txt -w 4 -S 1 -D 4096 -j -T 1
+    $ docker exec -it dc-client /bin/bash /entrypoint.sh --m="W" --S=30 --D=1024 --w=8 --T=1
 
 ### Running the benchmark ###
 
 To determine the maximum throughput while running the workload with eight client threads,
-200 TCP/IP connections, and a get/set ratio of 0.8, use the following command:
+200 TCP/IP connections, and a get/set ratio of 0.8, use the following command. `TH` refers to the _throughput_ mode of operation.
 
-    $ ./loader -a ../twitter_dataset/twitter_dataset_30x -s docker_servers.txt -g 0.8 -T 1 -c 200 -w 8
+    $ docker exec -it dc-client /bin/bash /entrypoint.sh --m="TH" --S=30 --g=0.8 --c=200 --w=8 --T=1 
 
-This command will run the benchmark with the maximum throughput; however, the QoS requirements will highly likely be violated. Once the maximum throughput is determined, you should run the benchmark using the following command:
+This command will run the benchmark with the maximum throughput; however, the QoS requirements will highly likely be violated. Once the maximum throughput is determined, you should run the benchmark using the following command. `RPS` means that the client container will keep the load at the given load (requests per second).   
 
-    $ ./loader -a ../twitter_dataset/twitter_dataset_30x -s docker_servers.txt -g 0.8 -T 1 -c 200 -w 8 -e -r rps
+    $ docker exec -it dc-client /bin/bash /entrypoint.sh --m="RPS" --S=30 --g=0.8 --c=200 --w=8 --T=1 --r=rps 
 
 where `rps` is 90% of the maximum number of requests per second achieved using the previous command. You should experiment with different values of `rps` to achieve the maximum throughput without violating the target QoS requirements.
 
-When you are done with benchmarking, just type `exit` to quit the client container.
-As the server containers are running as daemons, you have to stop them using `docker`:
-
-    $ docker stop dc-server1
+Note that the last two commands will continue forever if you do not stop or kill the command. For running the command for a given amount of time, you can use the timeout command. The following example will run the benchmark in the `RPS` mode for 20 seconds:
 
+    $ docker exec -it dc-client timeout 20 /bin/bash /entrypoint.sh --m="RPS" --S=30 --g=0.8 --c=200 --w=8 --T=1 --r=100000 
 
 ## Important remarks ##
 - It takes several minutes for the server to reach a stable state.
 
-- The target QoS requires that 95% of the requests are serviced within 10ms.
+- The target QoS requires that 95% of the requests are serviced within 10 ms.
```

Comment:
I think the SLO for data caching should be 1ms. 10ms is roughly the time of reading from the disk, so it is not meaningful to caching in memory.